### PR TITLE
Use the python3 way to create virtualenv

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     $ git clone git@github.com:OCA/maintainers-tools.git
     $ cd maintainers-tools
-    $ virtualenv env
+    $ python3 -m venv env
```
Comment (sbidoul) [inline README.md:10]: Note that to get something really equivalent you'll need to add `env/bin/pip install -U pip setuptools wheel` to get the latest `pip` and `setuptools` and more importantly to add `wheel`. Indeed python's venv module does not install `wheel` and when it is absent, pip downgrades to legacy behaviour in several code paths.

Comment (dreispt) [reply README.md:10]: This means that `wheels` installation is also needed?

Comment (sbidoul) [reply README.md:10]: @dreispt it's not strictly necessary but when it's absent, pip falls back to legacy behaviours. So better take good habits.

**Comment by yvaucher** [comment]
@eLBati note that the README contains 2 places where the git clone and env are done:

You will want to sync the procedure here:
https://github.com/OCA/maintainer-tools/blob/master/README.md#developers

**Comment by sbidoul** [comment]
In any case I'd say it's still better to recommend virtualenv which gives better defaults for python packaging this days. Moreover python3 venv is not even installed by default on Debian and Ubuntu due to the unbundling of the python stdlib done by the Debian distro.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
     $ . env/bin/activate
     $ python setup.py install
```
Comment (simahawk) [inline README.md:12]: BTW I'd use `pip install -e .`

**Review by simahawk — COMMENTED** [review]
What do we do w/ this?

File: README.md (hunk 1 above)
Comment (sbidoul) [reply README.md:10]: Now that pip does not do `setup.py install` anymore, I'm now comfortable with recommending `python3 -m venv` followed by a pip upgrade.

**Comment by dreispt** [comment]
I think this can be closed.

**Pull request closed without merging**